cores/nRF5/wiring_analog_nRF52.c

```diff
@@ -33,38 +33,14 @@ static uint32_t saadcGain      = SAADC_CH_CONFIG_GAIN_Gain1_6;
 
 static bool saadcBurst = SAADC_CH_CONFIG_BURST_Disabled;
 
-#if 0 // Note: Adafruit use seperated HardwarePWM class
-#define PWM_COUNT 3
-
-static NRF_PWM_Type* pwms[PWM_COUNT] = {
-  NRF_PWM0,
-  NRF_PWM1,
-  NRF_PWM2
-};
-
-static uint32_t pwmChannelPins[PWM_COUNT] = {
-  0xFFFFFFFF,
-  0xFFFFFFFF,
-  0xFFFFFFFF
-};
-static uint16_t pwmChannelSequence[PWM_COUNT];
-#endif
+// Note: Adafruit use seperated HardwarePWM class
```

Comment:
Ok.  if you want, this could possibly be reverted.  I would have to regenerate the history of the branch, but it's possible.  That said, if there are changes in this section, I think git would know to ignore them during a merge....